packages/cli/src/credentials/credentials.controller.ts

```diff
@@ -79,8 +83,12 @@ export class CredentialsController {
 	}
 
 	@Get('/new')
-	async generateUniqueName(req: CredentialRequest.NewName) {
-		const requestedName = req.query.name ?? this.globalConfig.credentials.defaultName;
+	async generateUniqueName(
+		_req: unknown,
+		_res: unknown,
+		@Query query: GenerateCredentialNameRequestQuery,
+	) {
+		const requestedName = query.name ?? this.globalConfig.credentials.defaultName;
```

Comment:
looking at the frontend code, we seem to always be sending the `name` query param. should we update the typings to either make this mandatory, or move the entire default naming logic to the backend?
The current setup has fallback logic on the backend as well as the frontend, which IMO feels redundant.

Reply:
Agreed. I made it mandatory and removed the configuration option entirely. This should be safe to make since this is an internal API endpoint

Reply:
should we create a ticket to cleanup this code later?